app/javascript/components/sponsor_us/SponsorshipOption.jsx

```diff
@@ -2,23 +2,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Card from 'components/Card';
 
-const SponsorshipOption = ({ title, amount, selected }) => {
+const SponsorshipOption = ({ title, amount, selected, onClick }) => {
     return (
-        <Card className={`sponsorship-option ${selected ? 'selected' : null}`}>
-            <div className="flex flex-row justify-between">
-                <div className="sponsorship-option-radio flex flex-row items-center">
-                    <input type="radio" value={title} checked={selected} readOnly />
-                    <label htmlFor={title}>{title} Sponsor</label>
+        // eslint-disable-next-line jsx-a11y/label-has-associated-control
+        <label className="sponsorship-option-label">
```

Comment:
The biggest changes I made was moving the label from the inside of the component to the outermost element. I did this because putting the label outside of the radio button itself makes the entire element clickable and properly accessible.

EsLint seems a bit confused about this (hence the disable comment on the line above) -- I don't think I need to specify which input this label belongs to if the input is a child of the label.

Reply:
this looks fine to me, though you can also explore updating the depth attribute in the [rule details ](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md#case-my-label-and-input-components-are-custom-components)so that it can try to find the nested child

Reply:
Thanks so much for this link! I was trying to find something like this but did not do a good job Googling.